{{cookiecutter.plugin_name}}/pyproject.toml

```diff
@@ -111,6 +138,11 @@ skip-magic-trailing-comma = false
 # Like Black, automatically detect the appropriate line ending.
 line-ending = "auto"
 
+[tool.uv]
```

Comment:
This is not always working in MacOS (I think depending on the version) and Windows, and in Ubuntu Bernadette was also having the same issue. I saw all of this during the hackathon last week

Reply:
Are you sure they were not using just `pip` here? This is just the equivalent to passing the argument in the CLI.